.gitpod.yml

```diff
@@ -0,0 +1,4 @@
+# Gitpod config. For details, see https://www.gitpod.io/docs/config-gitpod-file.
+
+tasks:
+  - init: npm install && npm run check-links
```

Comment:
It might be good to also use `npm run serve` so that folks can have a dev version of the site built and running in their environment. But I'm also fine if that's not included.

Reply:
My preference would be to first commit this minimal version and then we can experiment and grow it over time. Thanks for the review and approval. Will merge this now as is.